docs/military/darwin.md

```diff
@@ -4,4 +4,185 @@
+
+!!! tip
+    If a VFR aircraft has not planned via a VFR route as above, use the phraseology: “ABC, cleared amended route VFR route 1, maintain A020, squawk 4512”
+
+#### Auto-Release Wedge
+
+The Darwin auto-release wedges extend from the departure end runway threshold, 30 degrees splayed from the centreline, to 7nm  
+
+Auto-release at Darwin is only available for VFR aircraft and low-level IFR aircraft.
+
+Tower may depart aircraft that conform with the above conditions within the auto-release wedge using the following headings:
+
+|Departure Type| Runway 11| Runway 29|
+|---|---|----|
+|VFR |H070V, H106V or H140V| H260V, H286V or H320V|
+|IFR (Visual Dep)| H070V, H106V, or H140V |H260V, H286V or H320V|
+|IFR (Radar)| H070, H106 or H140 |H260, H286 or H320|
+
+!!! tip
+    Tower shall provide visual separation between aircraft in the auto-release wedge and subsequent departures until the preceding departure has been maneuvered by TCU
+
+#### IFR Level Assignment
+
+Issue `F180` or `RFL`; whichever is lower, to aircraft cleared via an RNAV SID or aircraft with a wake turbulence category of medium or greater.  
+Issue `F180` or `RFL`; whichever is lower, to jet aircraft not cleared via an RNAV SID regardless of wake turbulence category.  
+Issue `A030` or `RFL`; whichever is lower, to aircraft not cleared via an RNAV SID with a wake turbulence category of light.  
+
+
+### DN TCU Internal
+
+Heads-up coordination is not required between DAW and DAE for:
+
+- Departures on a Procedural SID and assigned `F180`  
+- Arrivals on a STAR assigned `A060`  
+
+All other aircraft must be heads-up coordinated prior to the airspace boundary  
+
+!!! example
+    <span class="hotline">**DAW** -> **DAE**</span>: "via DN, CFH23"  
+    <span class="hotline">**DAE** -> **DAW**</span>: "CFH23, A090"   
+
+## Miscellaneous
+### Circuit Operations
+VFR aircraft that will operate only in ADCs airspace shall be assigned SSR code 0100  
+
+Circuit altitude will depend on the type of aircraft. Assign circuit altitudes for the following aircraft types:  
+a) MIL Jet: `A020`  
+b) Jet: `A015`  
+c) Non-jet: `A010`  
+d) Helo: `A010`  
+### IFR Training Area
+IFR aircraft requiring an area for training shall be cleared to operate in the North Eastern Training Area (NETA)  
+The NETA is defined as the area between DN 360R–060R from 15NM–30NM DN DME.  
+Aircraft are to be cleared to the NETA via the 030R outbound. Vertical limits are to be specified by DAW prior to issuing airways clearance  
+
+### Designated Fuel Dumping Area
+Other than in an emergency, the designated fuel jettison area is:  
+a) Over water in the Beagle Gulf  
+b) Between the 320 and 020 TACAN radials  
+c) `A060` or above  
+### Robertson Barracks
+Darwin TMA is not responsible for traffic or separation services within the A005 step to the east of Darwin. This step is designed for low-level VFR MIL helos to transit in/out of Robertson Barracks within the need for communication with Darwin TMA.
```

Comment:
Possible typo?
`Robertson Barracks within the need for communication with Darwin TMA`